docs/BreakingChanges/096 Profiling ASP.Net MVC4 apps can lead to Fatal Execution Engine Error.md

```diff
@@ -0,0 +1,25 @@
+## 96: Profiling ASP.Net MVC4 apps can lead to Fatal Execution Engine Error
```

Comment:
{
  "Id": "96",
  "Title": "Profiling ASP.Net MVC4 apps can lead to Fatal Execution Engine Error",
  "Details": "Profilers using NGEN /Profile assemblies may crash profiled ASP.NET MVC4 applications on startup with a 'Fatal Execution Engine Exception'",
  "Suggestion": "This issue is fixed in the .NET Framework 4.5.2. Alternatively, the profiler may avoid this issue by specifying `COR_PRF_DISABLE_ALL_NGEN_IMAGES` in its event mask.",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/793730/clr-profiler-crashes-any-asp-net-mvc4-app-on-startup-due-to-ngen-profile-images",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5.2",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis